### ISSUE TITLE
Adding * to eslintignore doesn't work

### ISSUE THREAD

**Issue by jayair** [serverless-stack/serverless-stack#67]
Need an option to disable ESLint and need to update the docs:

https://docs.serverless-stack.com/working-locally#linting-type-checking

**Comment by RyanHow** [comment]
I tried this in regards to #66 in an attempt to disable the linting, but it didn't have any impact

**Comment by jayair** [comment]
Yup, we changed the way to handle linting. We use the output from esbuild to figure out which files to lint. So this option doesn't work anymore. We'll add a separate option.

**Comment by thdxr** [comment]
We are transition this repo to https://github.com/sst/v2 and are closing this issue for now. If it is still relevant please reopen it there - sorry for the trouble
